command/create.go

```diff
@@ -25,7 +25,7 @@ import (
 	"regexp"
 )
 
-// Input struct presents input options for flyte command
+// Input struct presents input options for CreateIssue flyte command
```

Comment:
I would say don't add comments to the source code unless they re trying to explain something really super important and very complex.

I guess what might be more appropriate in this case is to rename `Input` -> `CreateIssueInput` or if you want to be super descriptive `CreateIssueCmdInput`

Reply:
Fixed